cmd/internal/cli/verify.go

```diff
@@ -141,44 +136,64 @@ var VerifyCmd = &cobra.Command{
 }
 
 func doVerifyCmd(ctx context.Context, cmd *cobra.Command, cpath, url string) {
+	id, isGroup, err := ensureImageFlags(cmd, cpath, sifDescID, sifGroupID, verifyAll)
+	if err != nil {
+		sylog.Fatalf("%s", err)
+	}
+
+	// Dont need to resolve remote endpoint.
+	if !localVerify {
+		handleVerifyFlags(cmd)
+	}
+
+	author, _, err := signing.Verify(ctx, cpath, url, id, isGroup, verifyAll, authToken, localVerify, jsonVerify)
```

Comment:
We pass `verifyAll` here - but we are also using it in the call to `ensureImageFlags` which returns `id, isGroup`. This is a bit confusing - as it is not immediately clear if `id, isGroup` are ignored, or affected when `verifyAll` is true.

Reply:
We pass `verifyAll` to `ensureImageFlags()` only to check that the `--sif-id`/`--group-id` are not used at the same time. Since we will need to check that for sign and verify, I just added that check to that function, just so theres not duplicate code, witch is what this PR is supposed to improve.